<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="DESKTOP-UUTUU8G" name="practice.TechUs.ContactPage" tests="4" failures="1" timestamp="11 Sep 2019 10:38:47 GMT" time="38.031" errors="0">
  <testcase name="pageTitleVerify" time="11.181" classname="practice.TechUs.ContactPage">
    <skipped/>
  </testcase> <!-- pageTitleVerify -->
  <testcase name="pageTitleVerify" time="8.206" classname="practice.TechUs.ContactPage">
    <skipped/>
  </testcase> <!-- pageTitleVerify -->
  <testcase name="pageTitleVerify" time="9.994" classname="practice.TechUs.ContactPage">
    <skipped/>
  </testcase> <!-- pageTitleVerify -->
  <testcase name="pageTitleVerify" time="8.650" classname="practice.TechUs.ContactPage">
    <failure type="java.lang.AssertionError" message="expected [Tell Us About Your Dream. Let&#039;s chat! - Tech.us1] but found [Tell Us About Your Dream. Let&#039;s chat! - Tech.us]">
      <![CDATA[java.lang.AssertionError: expected [Tell Us About Your Dream. Let's chat! - Tech.us1] but found [Tell Us About Your Dream. Let's chat! - Tech.us]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at resources.Base.menu(Base.java:79)
at practice.TechUs.ContactPage.pageTitleVerify(ContactPage.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- pageTitleVerify -->
</testsuite> <!-- practice.TechUs.ContactPage -->
